ui/DesignSystem/Component/Stats.svelte

```diff
@@ -1,13 +1,19 @@
 <script>
   import { Icon, Numeric } from "../Primitive";
 
-  export let stats = [
-    { icon: Icon.Commit, count: -1 },
-    { icon: Icon.Branch, count: -1 },
-    { icon: Icon.Member, count: -1 },
-  ];
+  export let stats = {
+    contributors: null,
+    branches: null,
+    commits: null,
+  };
```

Comment:
Instead of having this an entire object, each could be its own prop.

Reply:
ooo good idea